blog/electron-27-0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Electron 27.0.0
+date: 2023-10-10T00:00:00.000Z
+authors:
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-27-0
+---
+
+Electron 27.0.0 has been released! It includes upgrades to Chromium `118.0.5993.32`, V8 `11.8`, and Node.js `18.17.1`. Read below for more details!
```

Comment:
```suggestion
Electron 27.0.0 has been released! It includes upgrades to Chromium `118.0.5993.32`, V8 `11.8`, and Node.js `18.17.1`.
```

Personally I'd drop the "Read below for more details!", since it's implied in a blog post and doesn't (at least to me) add obviously useful information.